config/database.yml

```diff
@@ -1,38 +1,17 @@
+# You can theoretically use anything for the Database, but we default to MySQL
+# since this has been our setup since the beginning.
+
 default: &default
-  adapter: mysql2
-  encoding: utf8
-<% if ENV['COMPOSE'] %>
-  host: <%= ENV['PORTUS_DB_HOST'] %>
-  username: root
-  password: portus
-<% end %>
+  adapter:  <%= ENV["PORTUS_DB_ADAPTER"]  || "mysql2"    %>
 
-development:
-  <<: *default
-  database: portus_development
+  host:     <%= ENV['PORTUS_DB_HOST']     || "localhost" %>
+  port:     <%= ENV['PORTUS_DB_PORT']     || 0           %>
```

Comment:
Yeah. For mysql this means to use the configuration as defined in the config file.